koji-setup/mash.sh

```diff
@@ -5,10 +5,11 @@
 set -e
 . /etc/profile.d/proxy.sh || :
 
+KOJI_TAG="${KOJI_TAG:-"$1"}"
```

Comment:
```
"${1:?"Missing tag parameter"}"
```
Is probably a more accurate parameter expansion here.  If the service is enabled without a parameter, we want to indicate that this is required and have the script fail.